src/components/Form/Widgets/SelectWidget.tsx

```diff
@@ -58,11 +73,38 @@ export const SelectWidget = ({
 				}}
 				disabled={disabled}
 				options={selectOptions}
-				isOptionEqualToValue={(option, value) => option.value === value.value}
-				getOptionLabel={(option) => option.label}
-				getOptionDisabled={(option) => option.disabled}
+				isOptionEqualToValue={(option, value) => isEqual(option, value)}
+				getOptionLabel={(option) =>
+					Array.isArray(option)
+						? option.map((o) => o.label).join(', ')
+						: option.label
+				}
+				getOptionDisabled={(option) =>
+					Array.isArray(option)
+						? option.every((o) => o.disabled)
```

Comment:
I'm confused here. To see if 1 option is disabled, we need to check if all options are disabled?

Reply:
This should only apply to 1 option at a time. I don't think it's ever for multiple options, so I expect `option.disabled` to be fine 🤔

Reply:
I can agree on the fact that 1 is enough but option.disabled is not something you can do, please che the typing I've pested as a comment